UnityProject/Assets/Scripts/Items/Others/AdminHandTeleporter.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using UnityEngine; 
+
+/// <summary>
+/// Simple hand teleporter for convenience of admins.
+/// </summary>
+public class AdminHandTeleporter : MonoBehaviour, IInteractable<AimApply>
```

Comment:
Could you change this to `ICheckedInteractable` and add the Default Interaction checks

Reply:
well it is an admin tool after all

Reply:
The  Default Interaction checks stop players hacking as it check valid distance and inventory stuff